src/main/java/Bert.java

```diff
@@ -0,0 +1,354 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.lang.StringBuilder;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+
+/**
+ * A chatbot named Bert that keeps track of a task list.
+ */
+public class Bert {
+    private static final String BOT_NAME = "Bert";
+    private static final String FILE_PATH = "./data/bert.txt";
+    private static ArrayList<Task> al = new ArrayList<>();
+
+    /**
+     * Introduces itself.
+     */
+    private static void introduce() {
+        System.out.println(
+                "____________________________________________________________\n" +
```

Comment:
Looks that the underbar line is being reused. Would it be simpler to abstract this such as having a UI do it?